webpack.config.js

```diff
@@ -3,6 +3,13 @@ const webpack = require("webpack");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const VueLoaderPlugin = require("vue-loader/lib/plugin");
 const CopyPlugin = require("copy-webpack-plugin");
+const packageJson = require("./package.json");
+const version = packageJson.version || 0;
```

Comment:
```suggestion
const version = packageJson.version ?? 0;
```

In newer code, always use `??`, the nullish coalescing operator, instead of relying on the `number`-to-`bool` coercion semantics of `||`.

Reply:
Right, make perfect sense